content/en/docs/kubernetes/collector/components.md

```diff
@@ -13,15 +13,167 @@ enhancing it.
 
 Important Components:
 
-- [Filelog Receiver](#filelog-receiver) - used to collect Kubernetes logs and
+- [Kubernetes Attributes Processor](#kubernetes-attributes-processor) - adds
+  Kubernetes metadata to incoming telemetry.
+- [Filelog Receiver](#filelog-receiver) - collects Kubernetes logs and
   application logs written to stdout/stderr.
 
 For application traces, metrics, or logs, we recommend the
 [OTLP receiver](https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/otlpreceiver),
 but any receiver that fits your data is appropriate.
 
+## Kubernetes Attributes Processor
+
+| Deployment Pattern   | Usable |
+| -------------------- | ------ |
+| DaemonSet (Agent)    | Yes    |
+| Deployment (Gateway) | Yes    |
+| Sidecar              | No     |
```

Comment:
It's odd to start a section with a table. Can this be moved, if only after the first paragraph?

Reply:
I was attempting to replicate the feeling of the table that start each collector component's readme: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/k8sattributesprocessor. I can move it if you want, but there is some nice familiarity between the 2 when it is at the top.

Reply:
Ok, let's leave this as is then 👍🏻.